docs/extensions/tutorials/_posts/1970-01-05-Installing3rdPartyPackages.md

```diff
@@ -0,0 +1,323 @@
+
+```javascript{12,17-27,38-62}
+#file app/screens/SwiperScreen.js
+import React, {
+  Component
+} from 'react';
+
+import {
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+import Swiper from 'react-native-swiper';
+
+export default class SwiperScreen extends Component {
+  render() {
+    return (
+      <Swiper style={styles.wrapper} showsButtons={true}>
+        <View style={styles.slide1}>
+          <Text style={styles.text}>Hello Swiper</Text>
+        </View>
+        <View style={styles.slide2}>
+          <Text style={styles.text}>Beautiful</Text>
+        </View>
+        <View style={styles.slide3}>
+          <Text style={styles.text}>And simple</Text>
+        </View>
+      </Swiper>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  text: {
+    color: '#FFFFFF',
+    fontSize: 30,
+    fontWeight: 'bold',
+  },
+  wrapper: {
+  },
+  slide1: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#9DD6EB',
+  },
+  slide2: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#97CAE5',
+  },
+  slide3: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#92BBD9',
+  }
+});
+```
+
+And there we go, we implemented `react-native-swiper` into our extension. Now we need to push our extension to the Shoutem server and install it onto an app so we can actually test it out. Continuing where we stopped in our terminal:
+
+```ShellSession
+$ cd ..
+$ shoutem push
+Uploading `Swiper Extension` extension to Shoutem...
+Success!
+$ shoutem install --new SwiperApp
+Extension installed
+See it in browser: `{{ site.shoutem.builderURL }}/app/{{ site.example.appId }}`
+```
+
+Opening the SwiperApp in the Builder will show us an app with no Screens, but since we just installed our Swiper Extension onto the app, we can just add the Swiper Screen to it by clicking on the + button next to Screens, going to the Custom category and selecting the Swiper Extension. As soon as it's loaded into the Main Navigation we can preview the app:
+
+<p class="image">
+<img src='{{ site.url }}/img/tutorials/installing-3rd-party-and-native-packages/swiper-in-preview.png'/>
+</p>
+
+## 2) Installing a Native Package
+
+When installing a 3rd party package that uses native functionalities we have to make sure we link the native dependencies it has. This is done using postlink scripts. As an example, we'll be making a QR Code reader that's going to display what the scanned QR code says. To scan a QR code we'll need to use the devices camera, which we'll get access to using react-native-camera, a 3rd party package for utilizing device cameras.
```

Comment:
- Format react-native-camera as \`react-native-camera`
- Link postinstall scripts to npm documnetation (or rather somewhere in the React Native, since you'll be using `rnpm`)